dll/win32/shell32/dialogs/folder_options.cpp

```diff
@@ -308,6 +304,20 @@ ShowFolderOptionsDialog(UINT Page, BOOL Async = FALSE)
 static VOID
 Options_RunDLLCommon(HWND hWnd, HINSTANCE hInst, int fOptions, DWORD nCmdShow)
 {
+    ACTCTXW ActCtx;
+    HANDLE hActCtx;
+    ULONG_PTR Cookie;
+    BOOL bActivated;
+
+    // Enable visual styles
+    ZeroMemory(&ActCtx, sizeof(ActCtx));
+    ActCtx.cbSize = sizeof(ActCtx);
+    ActCtx.dwFlags = ACTCTX_FLAG_HMODULE_VALID | ACTCTX_FLAG_RESOURCE_NAME_VALID;
+    ActCtx.lpResourceName = MAKEINTRESOURCEW(ISOLATIONAWARE_MANIFEST_RESOURCE_ID); // Default DLL manifest
+    ActCtx.hModule = shell32_hInstance;
+    hActCtx = CreateActCtxW(&ActCtx);
```

Comment:
Is this manual manifest activation also needed on Windows?

Reply:
Windows (XPSP3) calls explorer.exe to open the dialog and doesn't use rundll32.exe to host the dll.

Reply:
Tested on Win2k3. This is not a bug in our loader. Calling the folder options dialog from ReactOS shell32.dll on Win2k3 has the visual style issue as well.

Command: `rundll32.exe <path_to_reactos_shell32_dll>,Options_RunDLL 0`